contrib/perf/post-pr-comment.py

```
@@ -0,0 +1,274 @@
+
+    # Post all chunks to each PR
+    success_count = 0
+    total_posts = len(args.prs) * len(comment_chunks)
+
+    for pr_number in args.prs:
+        for i, chunk in enumerate(comment_chunks):
+            print(f"Posting part {i + 1}/{len(comment_chunks)} to PR #{pr_number}...", file=sys.stderr)
+
+            if post_comment(owner, repo, pr_number, chunk, token):
+                success_count += 1
+                print(f"  ✓ Posted successfully", file=sys.stderr)
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)